src/schema/v2/artworksForUser/helpers.ts

```diff
@@ -7,18 +7,15 @@ export const getNewForYouRecs = async (
   args: CursorPageable,
   context: ResolverContext
 ): Promise<string[]> => {
-  const { appToken, vortexGraphqlLoader, vortexGraphqlLoaderFactory } = context
-
-  const graphqlLoader =
-    vortexGraphqlLoader || vortexGraphqlLoaderFactory(appToken)
+  const { vortexGraphqlLoader } = context
```

Comment:
This section also read awkwardly, now it uses the pattern of same-named loaders where an access token 'with authentication' one taking precedence.

Reply:
Do you need to? The accepted pattern in MP is to use the same name loader in both, so that the 'with' authentication one takes precedence when an access token is present.

Reply:
I tried and only this way works. Not sure if it is related to how 'with' loader is created.